src/Dfe.EarlyYearsQualification.Mock/Content/MockQualificationsRepository.cs

```diff
@@ -16,6 +16,8 @@ public class MockQualificationsRepository : IQualificationsRepository
                                                                           AwardingOrganisations.Various, 3)),
                    "eyq-108" => await Task.FromResult(CreateQtsQualification("EYQ-108", "BTEC",
                                                                              AwardingOrganisations.Various, 6)),
+                   "eyq-115" => await Task.FromResult(CreateQualification("EYQ-115", "NCFE",
```

Comment:
Question, non-blocking: is it important that this new line isn't covered by a unit test?

...there's an explicit test for the default `EYQ-240` and there's `GetQualificationById_EYQ108_ReturnsExpectedDetails`, but no test for either `EYQ-250` or the new `EYQ-115`. It might be overkill, but presumably EYQ-115 was added specifically to exercise the dynamic text, so is there something important and unique about it that's worth baking in?

Reply:
Added the unit test now. Added it for 250 too as that was missing also